src/app/components/utilities/richtext.scss

```diff
@@ -412,6 +412,11 @@
       }
     }
 
+    // TODO: This is a temporary fix for a but in PL's dark mode implementation
+    table th {
+      background-color: var(--od-colour-surface-utility);
+    }
```

Comment:
Is this needed for future news only? Or legacy too?

Reply:
Only future, they seem to have a new table component with incomplete dark mode